src/functionalTest/resources/responses/response-caseworker-issue-sole-citizen-application-app2-overseas-about-to-submit.json

```diff
@@ -0,0 +1,142 @@
+{
+  "data": {
```

Comment:
is all this data needed for this specific callback? might be easier to manage if you just restrict it to the data used in the callback